helm-chart/azp-agent/templates/azp-agent-secret.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: azp-agent-secret
+  name: {{ .Values.azpAgentName | default "azp-agent" }}-secret
```

Comment:
>azpAgentName [](start = 19, length = 12)

Could you remind me what we plan to use for .Value.azpAgentName? It might be something we just adopt as the new default in the chart.

Reply:
I'm not sure what you're asking. 

What do we plan on putting in as the value? In my case it's `gsservices-build` and `gsservices-release`

If you mean the purpose of the field, it's to name the StatefulSet as well as propagate the indexed version to the ADO agent.

Reply:
Now I remember you mentioned you needed different secrets for build and release agents. I think your change is ok for now. Later we might need to do an overall review of naming across the chart which I think is not very customizable.

---
In reply to: [381617550](https://github.com/julioct/azure-pipelines-kubernetes-agents/pull/1#discussion_r381617550) [](ancestors = 381617550)